### PR TITLE
Weatherford debug (ready for review)

### PR DESCRIPTION
**Pull request by solderq35**

### CODE REVIEW

**Comment by solderq35** [comment]
documentation work in progress: https://github.com/OSU-Sustainability-Office/energy-dashboard/issues/191

**Comment by solderq35** [comment]
Again: Refer here for documentation on this PR: https://github.com/OSU-Sustainability-Office/energy-dashboard/issues/191
(thought it will be more organized if all documentation relating to campaigns stuff is on that issue).

File: src/store/chart_modifiers/line_bar/accumulated_real.js [hunk 1]
```diff
@@ -35,10 +35,22 @@ export default class LineAccumulatedModifier {
...
+    // array that stores keys for the resultDataObject (used for finding nearest valid keys for Weatherford)
+    let keysarray = Array.from(resultDataObject.keys())
     let returnData = []
     let delta = 1
     let startDate = (new Date((payload.dateStart) * 1000))
     let monthDays = 1
+
+    // Finds the nearest valid keys for a given building (mostly intended for correcting manual meter uploads, e.g. Weatherford.)
+    // Other buildings with automatic meter upload should have the same keys after this function is run.
+    function findClosest (array, num) {
```
Comment (MilanDonhowe) [inline src/store/chart_modifiers/line_bar/accumulated_real.js:48]: Excellent!

**Review by MilanDonhowe — APPROVED** [review]
Looks good!

**Pull request merged**